src/components/Stakeholders/Filter.js

```diff
@@ -120,14 +120,14 @@ function StakeholderFilter() {
   const defaultOrgList = [
     {
       id: ORGANIZATION_NOT_SET,
```

Comment:
We should change this const name to `ALL_ORGANIZATIONS` for consistency

Reply:
I meant rename `ORGANIZATION_NOT_SET` (in `FilterStakeholder`) to `ALL_ORGANIZATIONS` everywhere it's used.